packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -470,10 +471,41 @@ export default class Meetings extends WebexPlugin {
      * @param {String} [options.feedbackId] ID used for tracking
      * @param {String} [options.locusId]
      * @param {Stirng} [options.correlationId]
+     * @param {String} meetingId
      * @returns {String} feedback ID logs were submitted under
      */
-    uploadLogs(options = {}) {
-      return this.loggerRequest.uploadLogs(options);
+    uploadLogs(options = {}, meetingId) {
+      return this.loggerRequest.uploadLogs(options)
+        .then((uploadResult) => {
+          LoggerProxy.logger.info('Meetings:index#destroy --> Upload logs for meeting completed.', uploadResult);
+          Trigger.trigger(
+            this,
+            {
+              file: 'meetings',
+              function: 'destroy'
+            },
+            EVENT_TRIGGERS.MEETING_LOG_UPLOAD_SUCCESS,
+            {
+              meetingId,
```

Comment:
What do we want to do in the case where someone manually calls `meetings.uploadLogs` without a meeting id?

Reply:
its just needed to indicate to the customer , as its on the meetings instance